blog/2024-05-21-blog-status-of-lbse-in-webkit.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Update on the Layer Based SVG Engine (LBSE) in WebKit"
+author: rbuis
+permalink: /blog/status-of-lbse-in-webkit.html
+thumbnail: /assets/lbse-logo-wide.png
+preview: A look back at the last 7 months of development on the Layer Based SVG Engine (LBSE) in WebKit.
+---
+
+This blog entry gives an update we at <a href="https://www.igalia.com">Igalia</a> did on upstreaming and development of LBSE in WebKit in the last 7 months. For an explanation of
+what LBSE is and how it is related to WPE see this previous <a href="https://wpewebkit.org/blog/05-new-svg-engine.html">entry</a> as a refresher.
+
+<div style="float: right">
```

Comment:
Nit: Can we swap the order of the logos here? I think that would be more consistent with the order Igalia and Wix are mentioned in the post.